### ISSUE TITLE
weakdeps should also be deps

### ISSUE THREAD

**Issue by orso82** [jverzani/SpecialPolynomials.jl#76]
`[weakdeps]` should also be listed as `[deps]` in the [Project.toml](https://github.com/jverzani/SpecialPolynomials.jl/blob/eb807f31728f9e3c44b34b84ac9670942801d663/Project.toml#L15-L17) otherwise one gets nasty warnings:

```
┌ Warning: Error requiring `FastGaussQuadrature` from `SpecialPolynomials`
│   exception =
│    LoadError: ArgumentError: Package SpecialPolynomials does not have FastGaussQuadrature in its dependencies:
│    - You may have a partially installed environment. Try `Pkg.instantiate()`
│      to ensure all packages in the environment are installed.
│    - Or, if you have SpecialPolynomials checked out for development and have
│      added FastGaussQuadrature as a dependency but haven't updated your primary
│      environment's manifest file, try `Pkg.resolve()`.
│    - Otherwise you may need to report an issue with SpecialPolynomials
...
```

See how for example [Polynomials.jl](https://github.com/JuliaMath/Polynomials.jl/blob/master/Project.toml) handles this.

**Comment by jverzani** [comment]
Thanks. Will fix as soon as I get a chance.

**Comment by jverzani** [comment]
I believe this can be closed now with #77. Thanks again for taking the time to let me know.
